### DISCUSSION TITLE
Telegram Notifier - Send Photo

### DISCUSSION THREAD

**Discussion by Nsbx** [symfony/symfony#44344] [Q&A] [answered]
Hello :)

I recently used symfony notifier, precisely telegram notifier, and i see actually the component doesn't support sendPhoto. (https://core.telegram.org/bots/api#sendphoto)

Is this voluntary ?

I can imagine the goal is only to send "notification" so i prefers to ask before create a PR to support sendPhoto method.

Thank in advance

**Comment by Nsbx** [comment]
Up

**Comment by stof** [comment, marked as answer]
the goal of this bridge is indeed only to implement sending notifications through Telegram, not to be a PHP SDK for the Telegram API (Packagist has [several packages](https://packagist.org/?query=telegram) that are providing such SDK though).
The notifier component does not have any concept of sending photos, and so the Telegram bridge has no reason to use `sendPhoto`

**Reply by Nsbx** [reply]
I see, thanks for your response 😄
